src/app/pages/search-character/search-character.component.scss

```
@@ -0,0 +1,50 @@
+@import '/src/scss/colors';
+
+.main-page {
+  height: 80vh;
+
+  .header {
+    background-color: $white;
+    height: 20vh;
+  }
+
+  .presentation {
+    align-items: center;
+    background-image: 
+      linear-gradient(360deg, $cerulean 0%, $pear 100%), 
+      url('../../../assets/images/main-presentation-background.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    height: 15vh;
+    justify-content: center;
+    
+    .title {
+      color: $white;
+      font-size: 26px;
+      font-weight: 700;
+      line-height: 39px;
+      text-align: center;
+      text-shadow: 0px 2px 4px $black;
+    }
+  }
+
+  .input {
+    margin: -50px auto;
+    min-width: 585px;
+    width: 50%;
+
+    &::before {
+      content: '';
+      background: url('../../../assets/images/main-background.png');
```

- you could also use

```suggestion
      background: url('/assets/images/main-background.png');
```